GVRf/Framework/src/org/gearvrf/GVRViewManager.java

```diff
@@ -83,6 +86,8 @@
 class GVRViewManager extends GVRContext implements RotationSensorListener {
 
     private static final String TAG = Log.tag(GVRViewManager.class);
+    private static final boolean DEBUG_STATS = true;
```

Comment:
Let's keep all debug flags off even if overhead is small. We can turn if on whenever we need to collect it.
    protected static final boolean DEBUG_STATS = false; // keep it off